packages/autocomplete/index.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('completer.completer', () => {
     });
 
     it('is an exact match to one of shell completions', () => {
-      const i = 'db.bios.find({ field: { $exis';
+      const i = 'db.bios.find({ field: { $exists';
```

Comment:
I might be misunderstanding this, but wouldn’t the point of this test be that it completes `$exis` to `$exists`?

Reply:
lol yep, you're right!